pkgs/top-level/python-packages.nix

```diff
@@ -5708,7 +5708,11 @@ in {
 
   pymetno = callPackage ../development/python-modules/pymetno { };
 
+<<<<<<< HEAD
   pymitv = callPackage ../development/python-modules/pymitv { };
+=======
```

Comment:
You did not cleanly resolve the merge conflict.

Reply:
Looks like that the previous forced pull didn't pick-up the changes.